### ISSUE TITLE
Instructions for using it

### ISSUE THREAD

**Issue by diegopau** [ShendoXT/soleiledit#3]
Hi, I would like to use this but I don't know where to start, what sort of save slot format is this expecting? does it work with emulators?

**Comment by ShendoXT** [comment]
Yes it's meant to be used with the emulators. It's supposed to edit battery backed up save made by the emulator.
I believe I used a Gens emulator while I was developing the editor but I think any genesis/mega drive emu should suffice as long as it's storing it's battery saves in the raw format.

**Comment by diegopau** [comment]
Thanks for the reply! I tried with Gens 2.14 and I started to play the game, reached Rafflesia Training Grounds, saved the game and closed the emulator. This created a .srm file. Now I open SoleilEdit and go to File->Open and I open the file. After doing this I still see "This save slot is not active, changes won't be saved".
Is there any other step I should be doing?

**Comment by ShendoXT** [comment]
Yup, that's how it's supposed to be, just open the .srm file, edit to your liking and then save.

But there was an issue which I hope is now fixed. Save size detection was wrong, ups...
Try the newest 1.1 release and check if it's working now.

**Comment by diegopau** [comment]
Just tried and It works perfectly with version 1.1 !!
Thanks a lot!

**Comment by ShendoXT** [comment]
No worries,
enjoy.
